setup.py

```diff
@@ -10,7 +10,7 @@
       description="Finite Difference DSL for symbolic computation.",
       long_descritpion="""Devito is a new tool for performing
       optimised Finite Difference (FD) computation from high-level
-      symbolic problem definitions. Devito performs automated code
+      symbolic problem definitions. Devito performs automated codes
```

Comment:
I have no idea where this change comes from, don;t remember making it  :) Will put it back.